Jbarget/oxymore
# So/leo adef

### What changes have you made?
- all images have been replaced with the same images but without text 
- all images have been compressed

### Which issue(s) does this PR fix?

Fixes #365 
Fixes #10 

### Screenshots (if there are design changes)
<img width="227" alt="Screenshot 2021-01-28 at 15 13 53" src="https://user-images.githubusercontent.com/53219789/106158685-e7a2a600-617b-11eb-9621-3c3267941528.png">


### How to test
- Set the timer back to a date in the past and the content should display
- Check that there are 14 images
- There should be no text on any of the images
